configurer.go

```diff
@@ -21,6 +21,7 @@ type ClientConfigurer struct {
 	timeout      time.Duration
 	remoteBinary string
 	sshClient    *ssh.Client
+	preserve     bool
```

Comment:
perhaps a more descriptive field name can be used here or the field can be better documented.